.gitignore

```diff
@@ -0,0 +1,7 @@
+.DS_Store
+*.pyc
+*.egg-info
+*.swp
+tags
+node_modules
+__pycache__
```

Comment:
Unrelated. This list is kind of small. 

Update this in Framework. We can create a mix of github's python + JS template.